static/locales/en-US.yaml

```diff
@@ -269,6 +269,7 @@ History:
   Empty Search Message: There are no videos in your history that match your search
   Search bar placeholder: "Search in History"
   Case Sensitive Search: Case Sensitive Search
+  Sort By Date ASC: Sort By Date ASC
```

Comment:
**issue:** Related to the above, not a fan of this label, but hard to know of a better alternative without switching to `ft-select`. If you do do that, looking at the other sorting labels we have, more consistent ones would be `DateWatchedOldest: Earliest Watched First` and `DateWatchedNewest: Latest Watched First`

Reply:
I was also not a big fan of the label, but I also wasn't sure what to write, I'll look into ft-select and update the label accordingly (I like the Latest/Earliest watched first label more)

Reply:
I changed the sort by texts please take a look